packages/core/components/Modal/CodeSnippet/index.tsx

```diff
@@ -1,49 +1,61 @@
+// components/Modal/CodeSnippet/CodeSnippet.tsx
 import { DirectionalHint, TooltipHost } from "@fluentui/react";
 import * as React from "react";
-import { useSelector } from "react-redux";
 import SyntaxHighlighter from "react-syntax-highlighter";
 
-import { ModalProps } from "..";
 import BaseModal from "../BaseModal";
 import { PrimaryButton, TertiaryButton } from "../../Buttons";
-import { interaction } from "../../../state";
 
 import styles from "./CodeSnippet.module.css";
 
 const PYTHON_PANDAS_MINIMUM = "Python 3.8+ (pandas)";
 
+export interface CodeSnippetProps {
+    onDismiss: () => void;
+    code?: string;
+    setup?: string;
+    title?: string;
+}
+
 /**
- * Dialog meant to show the user a Code snippet
+ * Body-only variant, suitable for embedding inside other modals/layouts.
```

Comment:
I'm a little confused by this, is this being used anywhere independently as an import right now?